### ISSUE TITLE
Runtime error in 9.3.0

### ISSUE THREAD

**Issue by de-don** [vendure-ecommerce/ngx-translate-extract#62]
After updating to `9.3.0` from `9.2.1` I faced the issue:
```
 import pkg, { isPropertyAccessExpression, isCallExpression } from 'typescript';
                                                ^^^^^^^^^^^^^^^^
      SyntaxError: Named export 'isCallExpression' not found. The requested module 'typescript' is a CommonJS module, which may not support all module.exports as named exports.
      CommonJS modules can always be imported via the default export, for example using:
      
      import pkg from 'typescript';
      const { isPropertyAccessExpression, isCallExpression } = pkg;
      
          at ModuleJob._instantiate (node:internal/modules/esm/module_job:146:21)
          at async ModuleJob.run (node:internal/modules/esm/module_job:229:5)
          at async ModuleLoader.import (node:internal/modules/esm/loader:473:24)
      
      Node.js v20.17.0
      
      Pass --verbose to see the stacktrace.
```

Versions:
```
"typescript": "5.4.5",
"@ngx-translate/core": "15.0.0",
"@vendure/ngx-translate-extract": "9.3.0",
```

**Comment by pmpak** [comment]
Hello, the issue has been resolved by #60 and will be available in the next release.

**Comment by michaelbromley** [comment]
v9.3.1 was just published 👍
